src/components/Action/constants.ts

```diff
@@ -0,0 +1,3 @@
+export const SIZES = { BIG: 'big', SMALL: 'small' };
```

Comment:
The correct way is to use typescript enums string union depeneding on how you want it used.